src/PKSim.Core/CoreConstants.cs

```diff
@@ -837,13 +834,10 @@ public static class Parameters
          public static readonly string MOLECULAR_WEIGHT = Constants.Parameters.MOL_WEIGHT;
          public static readonly string REFERENCE_CONCENTRATION = "Reference concentration";
          public static readonly string REL_EXP = "Relative expression";
-         public static readonly string NORM_SUFFIX = " (normalized)";
          public static readonly string REL_EXP_BLOOD_CELL = "Relative expression in blood cells";
-         public static readonly string REL_EXP_BLOOD_CELL_NORM = NormParameterFor(REL_EXP_BLOOD_CELL);
          public static readonly string REL_EXP_PLASMA = "Relative expression in plasma";
-         public static readonly string REL_EXP_PLASMA_NORM = NormParameterFor(REL_EXP_PLASMA);
          public static readonly string REL_EXP_VASC_ENDO = "Relative expression in vascular endothelium";
-         public static readonly string REL_EXP_VASC_ENDO_NORM = NormParameterFor(REL_EXP_VASC_ENDO);
+         public static readonly string NORM_SUFFIX = " (normalized)";
```

Comment:
NORM_SUFFIX probably not required anymore?

Reply:
yes. I'll have to move it to the conversion project. Right now it's still used.
I am wondering if I may have to keep it when using an OLD Individual.... or an OLD Simulation to filter those parameters out. Or we'll have to update a bunch of stuff when converting..to discuss